examples/fluids/meshes/cylinder.geo

```diff
@@ -0,0 +1,154 @@
+// Created Thursday, October 13, 2022.
+// Number of elements
+N = 40;
+Nx1 = N/2 + 1; Rx1 = 1.00;
+Nx2 = N/2 + 1; Rx2 = 1.00;
+Ny  = N/2 + 1; Ry = 2.00;
+Nb  = N/2 + 1; Rb = 0.9;
+Nc  = N/2 + 1; Rc = 1.00;
+
+zspan = 0.2;
+
+DefineConstant[
+  H = {4.5, Min .1, Max 4.5, Step .1,
+    Name "Parameters/Height"}
+];
+
+DefineConstant[
+  xL = {4.5, Min .1, Max 4.5, Step .1,
+    Name "Parameters/XL"}
+];
+
+DefineConstant[
+  xR = {15.5, Min .1, Max 15.5, Step .1,
+    Name "Parameters/XR"}
+];
+
+DefineConstant[
+  cyldiameter = {1.41421356, Min .1, Max 1.41421356, Step .1,
+    Name "Parameters/CylDiameter"}
+];
+
+DefineConstant[
+  resparam = {xL + xR, Min .1, Max xL + xR, Step .1,
+    Name "Parameters/ResParam"}
+];
+
+// Exterior corners
+Point(1) = {-xL, -H, 0, 1.0};
+Point(2) = {xL, -H, 0, 1.0};
+Point(3) = {xR, -H, 0, 1.0};
+Point(4) = {-xL, H, 0, 1.0};
+Point(5) = {xL, H, 0, 1.0};
+Point(6) = {xR, H, 0, 1.0};
```

Comment:
I think you can remove the fourth argument (characteristic length) because you aren't using a mesh generation algorithm that respects it.